hexl/include/hexl/experimental/seal/get-ntt.hpp

```diff
@@ -0,0 +1,54 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+#pragma once
+
+#include "hexl/experimental/seal/locks.hpp"
+#include "ntt/ntt-internal.hpp"
+
+namespace intel {
+namespace hexl {
+
+struct HashPair {
+  template <class T1, class T2>
+  std::size_t operator()(const std::pair<T1, T2>& p) const {
+    auto hash1 = std::hash<T1>{}(p.first);
+    auto hash2 = std::hash<T2>{}(p.second);
+    return hash_combine(hash1, hash2);
+  }
+
+  static std::size_t hash_combine(std::size_t lhs, std::size_t rhs) {
+    lhs ^= rhs + 0x9e3779b9 + (lhs << 6) + (lhs >> 2);
```

Comment:
Can we add a comment here saying this is based on Golden Ratio Hashing with seeds?